lib/translate/BUILD

```diff
@@ -0,0 +1,11 @@
+cc_library(
+  name = "translator_lib",
+  srcs = ["common.cc", "status_translator.cc"],
```

Comment:
A better approach for the BUILD file would be to create a cc_library target for each translator component; in this case, one `status_translator` target, and one `common` target, as well as a target for each other command (inquiry, read, etc). Then we would have a translator target which brings everything together.

Something like this:

```
cc_library(
  name = "common",
  hdrs = ["common.h"],
  srcs = ["common.cc"],
  deps = [
      "@com_google_absl//absl/base",
  ],
)

cc_library(
  name = "status_translator",
  hdrs = ["status_translator.h"],
  srcs = ["status_translator.cc"],
  deps = [
      ":common"
      "//third_party/spdk_defs:nvme_defs_lib",
      "//lib:scsi_defs_lib",
      "@com_google_absl//absl/base",
  ],
)
```

This approach allows modular development, helps keep build rules small and clean, and is in general easier to follow.

@eyhhuang @harloff-g @JustinAWei @basimsahaf I would encourage you to talk about this and discuss the specifics of how you want to structure the BUILD file.

Reply:
Thanks for all your BUILD structure suggestions! Definitely a good idea to keep things modularized and clean, especially as the codebase expands. We'll split up the tests and libraries into their own targets

Reply:
https://github.com/googleinterns/scsi2nvme/pull/83